test/passes/optimize-instructions_all-features.txt

```diff
@@ -2791,17 +2824,20 @@
   )
   (drop
    (i32.eq
-    (local.get $x32)
-    (i32.const 10)
+    (i32.const 20)
+    (i32.add
+     (local.get $x32)
+     (i32.const 10)
+    )
```

Comment:
It turns out canonalization valid only for one top level, right? I was under the impression it doing for nested subexpressions as well

Reply:
Hmm, even after recursive canonalization consts still not swapped